src/prob/mld.jl

```diff
@@ -74,6 +80,80 @@ function build_mc_mld(pm::_PM.AbstractPowerModel)
 end
 
 
+"Multinetwork load shedding problem including storage (snap-shot)"
+function build_mn_mc_mld(pm::_PM.AbstractPowerModel)
```

Comment:
@pseudocubic and @tasseff, I suggest we have a quick meeting early next week to sync on the scope of MLD formulations in this package.

Based on the last discussion @pseudocubic and I had in Jan/Feb this year, I thought this formulation was meant to be a minimalist one (that is, only relaxed load indicator variables would be included).  What we have here is more like a full restoration formulation.  This is surely not what @tasseff needs for PWM and I think It might also be better place in another location, like PMR.